app/presenters/setting_presenter.rb

```diff
@@ -51,6 +45,13 @@ def id
     name
   end
 
+  def parse_and_update_value(value)
+    Foreman::Logging.logger('app').warn("Setting #{name} has no database value and can not be updated") and return unless db_model
+    db_model.parse_string_value(value)
+    res = db_model.save
+    res
+  end
```

Comment:
I did not figure how to get rid of this method, because we need something to proxy the errors in `Api::V2::SettingsController#update`